.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Test
```

Comment:
I think it's also important to migrate the nightly run of the restart recovery suite:
https://github.com/vmware/concord-bft/blob/master/.github/workflows/restart_recovery_suite.yml